remoteit/desktop
# [DESK-1135] send feedback

### Description

send feedback UI

### Test plan



1. Sign in
2. Go to lateral menu
3. Click on `Send Feedback`
4. You should see a modal for feedback

### Ticket
https://remoteit.atlassian.net/browse/DESK-1135
